src/shared/components/SearchInput/SearchInput.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import debounce from 'lodash.debounce';
+
+import Input from '../Input/Input';
+import Button from '../Button/Button';
+
+import './styles.scss';
+
+const places: string[] = ['Anglia', 'Polska', 'Hiszpania', 'Włochy'];
+
+function SearchInput(): JSX.Element {
+  const [inputValue, setInputValue] = useState<string>('');
```

Comment:
Read about type inference: https://www.typescriptlang.org/docs/handbook/type-inference.html
You do not need to add `<string>` if you have initialised with `''` - TS will know this is a string

Reply:
ok Fixed